provider/pkg/gen/_go-templates/helm/v3/chart.go

```diff
@@ -351,9 +351,23 @@ func (c *Chart) GetResource(gvk, name, namespace string) pulumi.AnyOutput {
 	if len(namespace) > 0 && namespace != "default" {
 		id = fmt.Sprintf("%s/%s", namespace, name)
 	}
+
 	key := fmt.Sprintf("%s::%s", gvk, id)
+
```

Comment:
Maybe more of a nit. It seems like we're already trying to account for the default namespace here -- could we also set `key = fmt.Sprintf("%s::default/%s", gvk, name` here instead of below?

Reply:
This is the format for explicit namespaces. It is in the form of `gvk::namespace/name`.
For the default namespace, we could have `gvk::default/name` or `gvk::name`. Notice the potential lack of `default` here. So there's 2 possible keys for an object in the default namespace, hence the fallback in the below lines.